assets/xml/objects/gameplay_keep.xml

```diff
@@ -1,369 +1,951 @@
+        <Texture Name="gEffectFireFootprint2Tex" OutName="effect_fire_footprint_2" Format="ia8" Width="32" Height="32" Offset="0x0001A620"/>
+        <Texture Name="gEffectFireFootprint3Tex" OutName="effect_fire_footprint_3" Format="ia8" Width="32" Height="32" Offset="0x0001AA20"/>
+        <Texture Name="gEffectFireFootprint4Tex" OutName="effect_fire_footprint_4" Format="ia8" Width="32" Height="32" Offset="0x0001AE20"/>
+        <Texture Name="gEffectFireFootprint5Tex" OutName="effect_fire_footprint_5" Format="ia8" Width="32" Height="32" Offset="0x0001B220"/>
+        <Texture Name="gEffectFireFootprint6Tex" OutName="effect_fire_footprint_6" Format="ia8" Width="32" Height="32" Offset="0x0001B620"/>
+        <Texture Name="gEffectFireFootprint7Tex" OutName="effect_fire_footprint_7" Format="ia8" Width="32" Height="32" Offset="0x0001BA20"/>
+        <Texture Name="gEffectFireFootprint8Tex" OutName="effect_fire_footprint_8" Format="ia8" Width="32" Height="32" Offset="0x0001BE20"/>
+        <DList Name="gEffectFireFootprintDL" Offset="0x0001C220"/>
+        <Texture Name="gEffectUnusedBubbles1Tex" OutName="effect_unused_bubbles_1" Format="ia8" Width="32" Height="32" Offset="0x0001C2C0"/>
+        <Texture Name="gEffectUnusedBubbles2Tex" OutName="effect_unused_bubbles_2" Format="ia8" Width="32" Height="32" Offset="0x0001C6C0"/>
+        <Texture Name="gEffectUnusedBubbles3Tex" OutName="effect_unused_bubbles_3" Format="ia8" Width="32" Height="32" Offset="0x0001CAC0"/>
+        <Texture Name="gEffectUnusedBubbles4Tex" OutName="effect_unused_bubbles_4" Format="ia8" Width="32" Height="32" Offset="0x0001CEC0"/>
+        <Texture Name="gEffectUnusedBubbles5Tex" OutName="effect_unused_bubbles_5" Format="ia8" Width="32" Height="32" Offset="0x0001D2C0"/>
+        <Texture Name="gEffectUnusedBubbles6Tex" OutName="effect_unused_bubbles_6" Format="ia8" Width="32" Height="32" Offset="0x0001D6C0"/>
+        <Texture Name="gEffectUnusedBubbles7Tex" OutName="effect_unused_bubbles_7" Format="ia8" Width="32" Height="32" Offset="0x0001DAC0"/>
+        <Texture Name="gEffectUnusedBubbles8Tex" OutName="effect_unused_bubbles_8" Format="ia8" Width="32" Height="32" Offset="0x0001DEC0"/>
+        <DList Name="gEffectUnusedBubblesDL" Offset="0x0001E2C0"/>
+        <Texture Name="gUnknownCircle5Tex" OutName="unknown_circle_5" Format="i8" Width="64" Height="64" Offset="0x0001E370"/>
+        <Texture Name="gEffectDamageParticles1Tex" OutName="effect_damage_particles_1" Format="i4" Width="32" Height="24" Offset="0x0001F370"/>
+        <Texture Name="gEffectDamageParticles2Tex" OutName="effect_damage_particles_2" Format="i4" Width="32" Height="24" Offset="0x0001F4F0"/>
+        <Texture Name="gEffectDamageParticles3Tex" OutName="effect_damage_particles_3" Format="i4" Width="32" Height="24" Offset="0x0001F670"/>
+        <Texture Name="gEffectDamageParticles4Tex" OutName="effect_damage_particles_4" Format="i4" Width="32" Height="24" Offset="0x0001F7F0"/>
```

Comment:
yeah i was thinking of it in a more literal sense, i don't know if hit mark is that good though since it implies that it like leaves marks on link or something, do you have any other ideas?

Reply:
not for now. if we decide to rename it we will rename the actual effect as well. So they can both be changed at the same time